ouroboros-consensus/docs/Contributing.md

```diff
@@ -1,5 +1,35 @@
-TODO eg git history goals etc
+# How to contribute to the Consensus Layer
 
-See the
-https://github.com/input-output-hk/ouroboros-network/ouroboros-consensus/docs/StyleGuide.md
-document.
+See the follow documents.
+
+  * this repository's [README][repo-README]; it links to design documents
+  * Consensus [style guide][consensus-style]
+  * Consensus [git process][consensus-git]
+  * Consensus [onboarding][consensus-onboarding]
+  * TODO quick-start guide
+
+Our primary neighbors are the upstream [cardano-ledger-specs] and the downstream
```

Comment:
this line is not needed in `Contributing.md` - people who read it most likely are aware of that fact. It brings unnecessary noise IMHO.

Reply:
OK. I'll yield to your more recent onboarding experience. Mine was focused on testing, so this wasn't obvious to me; I can believe that was not the normal. 👍